core/server/data/meta/index.js

```diff
@@ -66,7 +66,7 @@ function getMetaData(data, root) {
     if (settingsCache.get('logo')) {
         metaData.blog.logo.url = utils.url.urlFor('image', {image: settingsCache.get('logo')}, true);
     } else {
-        metaData.blog.logo.url = utils.url.urlJoin(utils.url.urlFor('admin'), 'img/ghosticon.jpg');
+        metaData.blog.logo.url = utils.url.urlJoin(utils.url.urlFor('admin'), 'assets/img/ghosticon.jpg');
```

Comment:
It is correct, that we're not using this image in Ghost Admin anymore, **but** we always want to show the default favicon when being in Admin, which is not correct at the moment (we're using the current favicon, which could be the uploaded one). Therefore, we'd either need to upload a default Ghost favicon in `/assets/img` for Admin, or we have to point to the favicon in our `/shared` folder?